instrumentation/opentelemetry-instrumentation-pymongo/tests/test_pymongo.py

```diff
@@ -57,22 +57,22 @@ def test_started(self):
         self.assertIs(span.kind, trace_api.SpanKind.CLIENT)
         self.assertEqual(span.name, "mongodb.command_name.find")
         self.assertEqual(span.attributes["component"], "mongodb")
-        self.assertEqual(span.attributes["db.type"], "mongodb")
-        self.assertEqual(span.attributes["db.instance"], "database_name")
+        self.assertEqual(span.attributes["db.system"], "mongodb")
+        self.assertEqual(span.attributes["db.name"], "database_name")
         self.assertEqual(span.attributes["db.statement"], "command_name find")
         self.assertEqual(span.attributes["net.peer.name"], "test.com")
         self.assertEqual(span.attributes["net.peer.port"], "1234")
         self.assertEqual(
-            span.attributes["db.mongo.operation_id"], "operation_id"
+            span.attributes["db.mongodb.operation_id"], "operation_id"
         )
         self.assertEqual(
-            span.attributes["db.mongo.request_id"], "test_request_id"
+            span.attributes["db.mongodb.request_id"], "test_request_id"
         )
 
-        self.assertEqual(span.attributes["db.mongo.filter"], "filter")
-        self.assertEqual(span.attributes["db.mongo.sort"], "sort")
-        self.assertEqual(span.attributes["db.mongo.limit"], "limit")
-        self.assertEqual(span.attributes["db.mongo.pipeline"], "pipeline")
+        self.assertEqual(span.attributes["db.mongodb.filter"], "filter")
```

Comment:
I don't see these in semantic conventions as well. What should these be?

Reply:
Same as above. If a database package adds a call level attributes to specific tech, it should be in the format of `db.{DBMS}.{attr}:{val}`.

Reply:
MongoDB has an interesting way of building the query, relying on function calls rather than a string containing the full statement.

The specification has a [convention for the statement](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#call-level-attributes), so if we followed the spec it may make more sense to set the raw query in the string instead, rather than put the sub-components of the statement in individual attributes.